src/commands/test/run/manifest.ts

```diff
@@ -33,4 +42,33 @@ export default class RunManifestTestsCommand extends RunTestsCommand {
   protected getSourceRootDir() {
     return path.dirname(this.manifestPath);
   }
+
+  protected async mergeTestArtifacts(): Promise<void> {
+    if (!this.outputXmlName) {
+      return;
+    }
+
+    const files: string[] = await walk(this.testOutputDir);
+    let xmlUtil: XmlUtil = null;
+    files.every((file: string) => {
+      if (path.extname(file) === ".zip") {
```

Comment:
@krukow 
I don't like this place.
Is it possible to have few archives inside test artifacts? In this case the behaviour is incorrect. If the correct archive is not first we get an exception `Unexpected reports type`. Also we get this exception in case when if artifacts don't contain any archive.

So with current realization we'll never get the exception:
> "test-output-dir" doesn't contain any mergeable test results (.zip archives containing .xml documents)

Reply:
@erychagov I'd probably write it so that it expects exactly one `.zip` file when `this.outputXmlName` is set. It ignores all other files, expects exactly one .zip file which is then merged. Raises errors if there is fewer or more than one?

Reply:
@krukow 
The code raises the error `Unexpected reports type` if the first (or single) archive has an incorrect name.
If there is not any archive the code raises the error `"test-output-dir" doesn't contain any mergeable test results (.zip archives containing .xml documents)`. I was wrong when I said `we'll never get the exception`

> I'd probably write it so that it expects exactly one .zip file

Ok, I'll do it